src/commands/ext-install.ts

```diff
@@ -302,8 +305,10 @@ export default new Command("ext:install [extensionName]")
     // If the user types in URL, or a local path (prefixed with ~/, ../, or ./), install from local/URL source.
     // Otherwise, treat the input as an extension reference and proceed with reference-based installation.
     if (isLocalOrURLPath(extensionName)) {
+      track("ext:install", "Install by Source");
       source = await infoInstallBySource(projectId, extensionName);
     } else {
+      track("ext:install", "Install by Extension Ref");
```

Comment:
Let's use something other than 'ext:install' for the action here - maybe 'Extension Installed'?. The default command tracking uses that same action, so we'll be double counting installs if we do this.

Reply:
Also, this may be a bit hacky, but we could  combine these 2 metrics into one and track interactive vs non interactive in the duration field, by passing in 1 for interactive mode and 0 for noninteractive. This would let us see interactive vs noninteractive for each label.